### DISCUSSION TITLE
Query Suggestions - Exact Matching

### DISCUSSION THREAD

**Discussion by xMarcelinex** [microsoft-search/pnp-modern-search#3869] [Q&A]
Hi I'm currently trying out the inbuilt query suggestions, but I'm running into an issue. 

When I type in the word 'leave' the query suggestions returns the items containing the word 'leave' then, when it can't find anymore, returns words like 'leader' and 'leadership. Assuming it's doing some fuzzy match.

I'm hoping to find if there is a way I can set the query suggestion to only return suggestions for the word provided. Is there a way to edit any of the available properties/achieve this without setting up custom query suggestions?

**Comment by kasperbolarsen** [comment]
I am not sure that I understand the issue,  can you share the configuration of the search box web part?

**Reply by xMarcelinex** [reply]
Hi, 

Apologies, to clarify, when I type the word 'leave' into the search box, the query suggestions return matches with leave such as 'annual leave' 'sick leave', but it also returns items with 'lea' such as 'lead' and 'leadership'.

I'm not too experienced in working with the PnP modern search, do you know which page of the settings I might find the configuration of the search box web part? Would it be the JSON in the raw properties?

Thanks.

**Reply by kasperbolarsen** [reply]
Please see https://microsoft-search.github.io/pnp-modern-search/usage/search-box/

**Reply by xMarcelinex** [reply]
Thanks! I am familiar with these settings. 

I have query suggestions enabled, and I'm using the SharePoint Static Seach suggestions provider with a defined list of words, are there any settings here that I can use to restrict the search? Or would I need to resolve this via the Manage Query suggestions in SharePoint?
